### DISCUSSION TITLE
Gibberish autocomplete in python

### DISCUSSION THREAD

**Discussion by i-am-zaidali** [community/community#102661] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/65814383/afb0e79c-081f-44d6-a081-bd63432617cf)

Used ctrl+enter to receive suggestions for autocomplete and got this gibberish instead in multiple suggestions

**Comment by mw-xyz** [comment]
Sometimes Copilot does bug out like this. Keep in mind your version - you may want to downgrade to stable if you're not already on that, and keep in mind AI isn't perfect. You can always ctrl-Z :) Let me know if this helps
